modulefiles/hera.intel/fv3

```diff
@@ -15,13 +15,13 @@ setenv CMAKE_CXX_COMPILER mpiicpc
 setenv CMAKE_Fortran_COMPILER mpiifort
 setenv CMAKE_Platform hera.intel
 
-module use /scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/v1.0.0-beta1/modulefiles/stack
+module use /scratch2/NCEPDEV/nwprod/hpc-stack/test/modulefiles/stack
```

Comment:
Please use the same naming convention for hpc-stack installation (lib/hpc-stack/vX.Y.Z). hpc-stack repo must have a tag pointing to that exact version number.